### DISCUSSION TITLE
[인프라] k3s 설치

### DISCUSSION THREAD

**Discussion by publdaze** [BDD-CLUB/challenge-every-week-posting#104] [1. 개발 관련 글 모음]
할 게 너무 많은데 몸이 안 따라줘요....🥲

[노션링크](https://sparkling-radium-6f4.notion.site/k3s-52406abf168b451986f598327838fa76?pvs=4)

**Comment by shkisme** [comment]
후엉 ㅜㅜ 할 수 있다!! 화이팅이에용! 🔥🔥🔥

![02a4dbcf400b4d8902e932f266be86d2](https://github.com/amaran-th/every-week-posting-challenge/assets/92802207/1c67b04a-e764-47d8-933e-d0e2de2bebe1)
